src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -8839,7 +8842,12 @@ class CL0<T>{}
 " }, parseOptions: TestOptions.Regular.WithNullCheckingFeature());
 
             c.VerifyDiagnostics(
-                );
+                // (13,17): warning CS8619: Nullability of reference types in value of type 'CL0<string?>' doesn't match target type 'CL0<string>'.
```

Comment:
>// (13,17): warning CS8619: Nullability of reference types in value of type 'CL0<string?>' doesn't match target type 'CL0<string>'. [](start = 16, length = 131)

I would think that this is an explicit conversion, therefore should be no warnings.